Dockerfile

```diff
@@ -1,39 +1,58 @@
-FROM alpine:latest
+FROM centos:latest
 LABEL maintainer="OSC"
 
 # Set language to avoid bugs that sometimes appear
 ENV LANG en_US.UTF-8
 
 # Set up requirements
-RUN apk upgrade --update \
-    && apk --no-cache add \
-          python3 \
-          py3-pip \
-          make \
-          openjdk8-jre \
-          ttf-dejavu \
-          graphviz
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash - \
+    && yum -y install \	   
+       openjdk8-jre \
+       ttf-dejavu \
+       graphviz \
+       alsa-lib \
+       gtk3 \
+       libXScrnSaver \
+       libappindicator \
+       libnotify \
+       git \
+       wget \
+       openssl \
+       nodejs \
+       npm \
+       which \
+       gcc-c++ make \
+       epel-release \
+    && yum clean all \
+    && rm -rf /var/cache/yum
 
 # Install PlantUML
-RUN apk add --no-cache --virtual .ssl-deps \
-      openssl \
-      ca-certificates \
-    && mkdir -p /opt \
+RUN mkdir -p /opt \
     && wget -O "/opt/plantuml.jar" "https://sourceforge.net/projects/plantuml/files/plantuml.jar" \
     && printf '#!/bin/sh -e\njava -jar /opt/plantuml.jar "$@"' > /usr/local/bin/plantuml \
-    && chmod 755 /usr/local/bin/plantuml \
-    && apk del .ssl-deps
+    && chmod 755 /usr/local/bin/plantuml 
 
 # Install Sphinx and extras
 ADD Pipfile* /tmp/
 
-RUN pip3 install pipenv=='2018.11.26' \
-  && cd /tmp \
-  && pipenv install --deploy --system \
-  && rm -rf /tmp/Pipfile*
+# Install pipenv
+RUN yum install -y python36-pip.noarch \
+    && pip3 install pipenv=='2018.11.26' \
+    && cd /tmp \
+    && pipenv install --deploy --system \ 
+    && rm -rf /tmp/Pipfile* \
+    && yum clean all \
+    && rm -rf /var/cache/yum
+
+# Install drawio-batch
+RUN git clone "https://github.com/languitar/drawio-batch.git" \
```

Comment:
Don't do get clone, which gets the entire history of the app. instead use a separate layer to `ADD https://github.com/languitar/drawio-batch/archive/10.6.3.tar.gz drawio-batch` or whatever the command is to get the result you want. I think ADD can actually unpack or unzip so you might choose the correct file.

Reply:
Got it, I will add a separate layer.

Reply:
There is also just using curl in the existing layer. That might actually be a better practice. I'm new to Docker, sorry.